.github/workflows/pr.yml

```diff
@@ -59,6 +59,12 @@ jobs:
           mkdir -p pr-previews/pr-${{ github.event.number }}
           cp -rf _site/* pr-previews/pr-${{ github.event.number }}
 
+      - name: Patch robots.txt
+        run: |
+          tee pr-previews/pr-${{ github.event.number }}/robots.txt <<EOF
```

Comment:
no need to add to a subdir,, the root file is good enough

Reply:
this is for the preview folder,
`pr-previews/pr-${{ github.event.number }}` should disallow everything.

the root will have the original robots.txt constructed from the `njk` template allowing everything besides the `pr-previews` folder.

Reply:
https://www.portent.com/blog/seo/robots-txt-mistakes-best-uses-guide.htm

yeah but there should be only one robots.txt per webroot

Reply:
cool - then in that case the one in the preview is redundant - I'll remove it.
:-)

Reply:
fixed.